CSS/style.css

```diff
@@ -1,3 +1,47 @@
 @import url("./basic-styling/color.css");
 @import url("./basic-styling/reset.css");
 @import url("./basic-styling/typography.css");
+
+@import url("./Components/index.css");
+
+/* Components Css */
+.container {
+  min-height: 100vh;
+  width: 80%;
+  margin: auto;
+  border: 1px solid black;
+}
+
+.navbar {
+  width: 100%;
```

Comment:
comment and check if it remains full width even without width:100%

Reply:
have removed it